src/unitxt/operators.py

```diff
@@ -213,6 +213,103 @@ def process(
         return flatten_dict(instance, parent_key=self.parent_key, sep=self.sep)
 
 
+class ModelInputFormatter(StreamInstanceOperator):
+    r"""Generates the whole input to the model, from constant strings that are given as args, and from values found in specified fields of the instance.
+
+    WholeInputFormatter expects its input instance to have been processed by a Template StreamInstanceOperator
```

Comment:
```suggestion
    ModelInputFormatter expects its input instance to have been processed by a Template StreamInstanceOperator
```

Reply:
Hi @matanor ,  `system_prompt` no longer exists. At least from the few examples and tests that I tried, Indeed no need to have the instruction string taking that short ride on the recipe, but it better be passed as an input arg to the constructor of `ModelInputFormatter`.  @elronbandel asked that I treat instruction as a resident of the input instance.  I will speak with him about the reason for that.